web/angular-wallet/src/app/setup/account/account.service.ts

```diff
@@ -69,7 +80,7 @@ export class AccountService {
 
   public setAccountInfo({ name, description, feeNQT, deadline, pin, keys }: SetAccountInfoRequest): Promise<TransactionId> {
     const senderPrivateKey = decryptAES(keys.signPrivateKey, hashSHA256(pin));
```

Comment:
hmm, should be more comfortable... maybe we need to introduce a function in burstjs for this? wdyt?

Reply:
Yeah feels a bit cumbersome...